### PR TITLE
Feat/versioned docs

### PR DESCRIPTION
**Pull request by SheepFromHeaven**
Adding docusaurus and removing docsify

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*/node_modules
```
Comment (tiagomapmarques) [inline .dockerignore]: Is the `*` necessary?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM node:8.11.4
+
+WORKDIR /app/website
+
+EXPOSE 3000 35729
+COPY ./docs /app/docs
+COPY ./website /app/website
+RUN yarn install
```
Comment (tiagomapmarques) [inline Dockerfile]: Does docusaurus require `yarn` or can we use npm?

File: docs/quick-start.md [hunk 3]
```diff
@@ -7,12 +10,6 @@ You can install the biotope element in your project using npm
 npm install @biotope/element --save
 ```
 
-or yarn
```
Comment (tiagomapmarques) [inline docs/quick-start.md]: We should keep this, even if we use `npm` ourselves.

File: package.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
     "build": "cpy src/*.scss lib && rollup -c",
     "watch": "cpy src/*.scss lib && rollup -c -w",
     "release": "np",
-    "docs": "docsify serve docs",
+    "docs": "cd website && npm start",
```
Comment (tiagomapmarques) [inline package.json]: Now I'm confused :/
Does docusaurus use `yarn` or `npm`?

File: .dockerignore (hunk 1 above)
Comment (SheepFromHeaven) [reply .dockerignore]: That's autogenerated

File: Dockerfile (hunk 2 above)
Comment (SheepFromHeaven) [reply Dockerfile]: That's autogenerated

File: package.json (hunk 4 above)
Comment (SheepFromHeaven) [reply package.json]: Docusaurus uses yarn

Comment (SheepFromHeaven) [reply package.json]: but this is our script

Comment (tiagomapmarques) [reply package.json]: Thanks for explaining

**Pull request merged**